binary_transparency/firmware/cmd/ft_personality/internal/http/server_test.go

```diff
@@ -80,6 +81,21 @@ func TestRoot(t *testing.T) {
 }
 
 func TestAddFirmware(t *testing.T) {
+	st, err := base64.StdEncoding.DecodeString("eyJEZXZpY2VJRCI6IlRhbGtpZVRvYXN0ZXIiLCJGaXJtd2FyZVJldmlzaW9uIjoxLCJGaXJtd2FyZUltYWdlU0hBNTEyIjoiMTRxN0JVSnphR1g1UndSU0ZnbkNNTnJBT2k4Mm5RUTZ3aExXa3p1UlFRNEdPWjQzK2NYTWlFTnFNWE56TU1ISTdNc3NMNTgzVFdMM0ZrTXFNdFVQckE9PSIsIkV4cGVjdGVkRmlybXdhcmVNZWFzdXJlbWVudCI6IiIsIkJ1aWxkVGltZXN0YW1wIjoiMjAyMC0xMS0xN1QxMzozMDoxNFoifQ==")
+	if err != nil {
+		t.Errorf("failed to decode metadata: %v", err)
+	}
+	sign, err := common.SignMessage(st)
+	statement := api.FirmwareStatement{Metadata: st, Signature: sign}
+	if err != nil {
+		t.Errorf("signing failed, bailing out!: %v", err)
```

Comment:
`Errorf` doesn't bail - it just marks the test as failed but allows the execution to continue, use `t.Fatalf` if this error means none of the rest of the test is valid.

(there might be other places in this file there this is applicable too)

Reply:
ACK